.github/workflows/test.yml

```diff
@@ -90,7 +90,10 @@ jobs:
         run: poetry config virtualenvs.in-project true
 
       - name: "Install additional libraries"
+        # TODO: Remove this workaround when the issue is fixed in Microsoft repo.
+        #       https://github.com/microsoft/linux-package-repositories/issues/130
         run: |
+          sudo rm /etc/apt/sources.list.d/microsoft-prod.list | true
```

Comment:
@matti-lamppu  , I came here accidentally from [upstream](https://github.com/microsoft/linux-package-repositories/issues/130). 
May I ask for the reason of piping into `true`?
```
sudo rm /etc/apt/sources.list.d/microsoft-prod.list || true
```
or
```
sudo rm -f /etc/apt/sources.list.d/microsoft-prod.list
```

Reply:
Hi! I added that as a safety just in case the file gets deleted. Piping to `true` suppresses the exit error code so that CI won't stop there.